.drone.yml

```
@@ -87,8 +87,8 @@ steps:
     commands:
       - cd environments
       - |
-        ADDRESS=$(grep -o '"contractAddress":"[^"]*' ../packages/smart-contract/ignition/deployments/chain-134/journal.jsonl | tail -1 | cut -d'"' -f4)
-        BLOCK=$(grep -o '"blockNumber":[0-9]*' ../packages/smart-contract/ignition/deployments/chain-134/journal.jsonl | tail -1 | cut -d':' -f2)
+        ADDRESS=$(jq -r '.contractAddress' ../packages/smart-contract/ignition/deployments/chain-134/journal.jsonl | tail -1)
```

There can't be more than 1:
```suggestion
        ADDRESS=$(jq -r '.contractAddress' ../packages/smart-contract/ignition/deployments/chain-134/journal.jsonl)
```

---

Same for others.

---

fix Commit: eaaba95706a5b374b30d257b71b4e27b5143f321